media.min.css

```diff
@@ -15,29 +15,29 @@
 @custom-media --stylus        (hover: none) and (pointer: fine);
 @custom-media --pointer       (hover) and (pointer: coarse);
 @custom-media --mouse         (hover) and (pointer: fine);
-@custom-media --xxs-only      (min-width: 0px) and (max-width: 240px);
-@custom-media --xxs-n-above   (min-width: 240px);
-@custom-media --xxs-n-below   (max-width: 239px);
+@custom-media --xxs-only       (min-width: 0px) and      (max-width: 240px);
+@custom-media --xxs-n-above (min-width: 240px);
+@custom-media --xxs-n-below (max-width: 239.98px);
```

Comment:
This avoids issues where fractional viewport widths don't have any styles. These cases are rare but do occur.